shim/src/repair/accessors.js

```diff
@@ -59,48 +60,46 @@ export function repairAccessors() {
     return obj;
   }
 
-  defineProperties(objectPrototype, {
```

Comment:
I think the original, with `defineProperties`, is better than the revision, with `definePropertyOrThrow`. Remember, we're only overcoming an irregularity with `Object.defineProperty` specifically, that `defineProperties` does not suffer from.

Reply:
I dropped this for a while since I wasn't able to find the irregularity in the specs:
```
19.1.2.4 Object.defineProperty

    If Type(O) is not Object, throw a TypeError exception.
    Let key be ? ToPropertyKey(P).
    Let desc be ? ToPropertyDescriptor(Attributes).
    Perform ? DefinePropertyOrThrow(O, key, desc).
    Return O. 
```
The spec matches the behaviour I see in node and all evergreen browsers. Was the issue restricted to WindowProxy?
https://esdiscuss.org/topic/figuring-out-the-behavior-of-windowproxy-in-the-face-of-non-configurable-properties